### PR TITLE
Reorganize modules with implicit namespace

### PR DESCRIPTION
**Pull request by eide**
Leave the top namespace, `origo`, empty to allow other packages to add their modules under it.

### CODE REVIEW

File: origo/sdk/status/status.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import logging
 
-from origo.sdk import SDK
+from origo.sdk.sdk import SDK
```
Comment (fredriv) [inline origo/sdk/status/status.py:3]: Should move SDK class up one level so we don't get multiple `.sdk` in module path?

**Review by fredriv — DISMISSED** [review]
I think we should move the `SDK` class up one level. Otherwise LGTM! :+1:

File: origo/sdk/status/status.py (hunk 1 above)
Comment (eide) [reply origo/sdk/status/status.py:3]: I think we can just add `from origo.sdk.sdk import SDK` in the top `__init__.py`, and that should allow `from origo.sdk import SDK` ...

I just got lost reading about what else you can put in `__init__` :grimacing:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## ?.?.?
+
+### Breaking
+
+* Modules reorganized to leave top level `origo` namespace empty, all modules
+  are now under `origo.sdk`. Ie. `from origo.data.upload import Upload` now
+  becomes `from origo.sdk.data.upload import Upload`.
+* This allows other packages to add their modules under the `origo` namespace.
```
Comment (simenheg) [inline CHANGELOG.md:8]: This sounds like it belongs to the previous bullet point?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+
```
Comment (simenheg) [inline CHANGELOG.md:10]: Other entries have just a single blank line between them.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# Installation
+
+`origo-sdk` is on PyPi: `pip install origo-sdk`
```
Comment (simenheg) [inline README.md:3]: PyPi => PyPI

File: origo/sdk/status/status.py (hunk 1 above)
Comment (simenheg) [reply origo/sdk/status/status.py:3]: Agreed that it should be `from origo.sdk import SDK`.

**Review by simenheg — DISMISSED** [review]
LGTM too! A few nitpick comments. :)

**Review by fredriv — APPROVED** [review]
LGTM! 🎉

**Review by fredriv — APPROVED** [review]

**Pull request merged**